website/allowed-subcategories.txt

```diff
@@ -5,7 +5,8 @@ App Configuration
 Catalog Management
 Certificate Manager
 Classic infrastructure
-Cloud Databases
+Cloudant
```

Comment:
The category you mentoined is Cloudant but in docs below its subcategory: "Cloudant Databases"

Reply:
If you want to seperate cloudant from Cloud Databases then please change the subcategory in exisitng reosurce/datasource of cloudan

Reply:
That's all right, I'll use "Cloudant Databases" for everything